### DISCUSSION TITLE
Order by id DESC

### DISCUSSION THREAD

**Discussion by rajeshisnepali** [dbeaver/dbeaver#14119] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/23610013/132936006-f9007bce-a15d-4c59-82f4-04f1d6975509.png)

I want to set a setting to order all my records by id `DESC`. Also, set a setting to view value in `JSON` format.

**Comment by ShadelessFox** [comment]
Hello @rajeshisnepali,

1. There's small arrow-like icon on the right half of the column's header. You can use it to change the order of records in this column. If you want this ordering to be persisted, you can press <kbd>Save filter settings</kbd> located in the upper right half of the data editor. See our wiki for more information: https://github.com/dbeaver/dbeaver/wiki/Data-Filters
1. It's scheduled for DBeaver 21.2.1. You can download an early access version to access this feature: https://dbeaver.io/files/ea

**Comment by rajeshisnepali** [comment]
> 1. There's small arrow-like icon

Yes, I use to do the same. Also I've configured shortuct key (`Ctrl+Num2`) to toggle the order and `Ctrl+.` to save the settings. But the problem is I've to do the same from every new table and that's so frustrating. 

> 2. It's scheduled for DBeaver 21.2.1

Waiting for 21.2.1. Currently running `Version 21.2.0.202109061154`

BTW, you said it's coming in 21.2.1 for ordering or JSON saving stuff. Li'l confused.

**Comment by ShadelessFox** [comment, marked as answer]
1. Indeed, you can't configure default ordering because it can cause performance issues if sorting is performed on server-side.
2. Now we save last used content viewer, in your case it's JSON viewer.
